packages/idempotency/tests/unit/persistence/DynamoDbPersistenceLayer.test.ts

```diff
@@ -604,7 +604,7 @@ describe('Class: DynamoDBPersistenceLayer', () => {
           id: dummyKey,
         }),
         UpdateExpression:
-          'SET #response_data = :response_data, #expiry = :expiry, #status = :status',
+          'SET #expiry = :expiry, #status = :status, #response_data = :response_data',
```

Comment:
is the reorder intentional?

Reply:
Yes, it's just in the the tests though. 

The reason why I needed to reorder them is that we are now composing the `UpdateExpression` in a different order and the `response_data` field is now last.